Stratis.Bitcoin/Features/BlockStore/BlockStoreFeature.cs

```diff
@@ -121,6 +124,8 @@ public static partial class IFullNodeBuilderExtensions
     {
         public static IFullNodeBuilder UseBlockStore(this IFullNodeBuilder fullNodeBuilder)
         {
+            LoggingConfiguration.RegisterFeatureNamespace<BlockStoreFeature>("db");
```

Comment:
Hmm I think db refers to to CoinView this is only BlockStore

Reply:
Compared to the code being replaced this should preserve the current behaviour - i.e. it does seem like "db" has been used in conjunction with the BlockStore feature up to now.